jquery.flot.touch.js

```diff
@@ -247,7 +247,13 @@
         }
 
         function isPinchEvent(e) {
-            return e.touches && e.touches.length === 2;
+            if (e.touches && e.touches.length >= 2) {
```

Comment:
Not necessarily for now, but some new tests have to be added for this code. Unfortunate there are only a few tests here for this plugin:
https://github.com/ni-kismet/engineering-flot/blob/master/tests/jquery.flot.touch.Test.js